vanilla_first_setup/utils/processor.py

```diff
@@ -135,19 +127,7 @@ def hide_first_setup(user: str=None):
         if user is None:
             user = os.environ.get("USER")
 
-        desktop_file = "/home/%s/.local/share/applications/org.vanillaos.FirstSetup.desktop" % user
         autostart_file = "/home/%s/.config/autostart/org.vanillaos.FirstSetup.desktop" % user
 
         if os.path.exists(autostart_file):
             os.remove(autostart_file)
-
-        with open(desktop_file, "w+") as f:
```

Comment:
This section creates a new desktop entry for the user when first-setup has finished to hide the icon from gnome-shell. This shouldn't be removed

Reply:
fixed in https://github.com/Vanilla-OS/first-setup/pull/170/commits/6954e05e5db453ba42ac66872771ea363bcd3373